src/client/java/daos-java/src/main/native/io_daos_DaosClient.c

```diff
@@ -363,44 +363,29 @@ Java_io_daos_DaosClient_createEventQueue(JNIEnv *env,
 	}
 	eq->nbrOfEvents = nbrOfEvents;
 	eq->eqhdl = eqhdl;
-	for (i = 0; i < nbrOfEvents; i++) {
-		eq->events[i] = (data_event_t *)malloc(sizeof(data_event_t));
-		if (eq->events[i] == NULL) {
-			char *msg = NULL;
-
-			asprintf(&msg, "Failed to allocate %d th event.",
-				 i);
-			rc = 1;
-			throw_base(env, msg, rc, 1, 0);
-			goto fail;
-		}
-		rc = daos_event_init(&eq->events[i]->event, eqhdl, NULL);
-		if (rc) {
+	for (; i < nbrOfEvents; i++) {
```

Comment:
minor, but style wise it's always better to have the initialization of the counter here.

Reply:
thanks